### ISSUE TITLE
Launch site Aug 9

### ISSUE THREAD

**Issue by kristinashu** [mozilla/ihr-landing-site#9]
@kaodro's timeline from https://github.com/MozillaFoundation/mofo-devops/issues/519

July 26 - Aug 9 max. Aug 15 (exact date is being negotiated): Translation of the website on staging (Kasia)
Aug 3: Updating legal notice in feedback form (Kasia)
Approx. Aug 4 (exact date TBD by Solana): blogpost on Indicators incl. image (mozilla/internet-health-report#36) added to staging (Solana)
Approx. Aug 9 or Aug 15 (exact date TBD soon) push Website to production (Cade, Kristina)
Directly before pushing to production: Adjusting old Report to fit the new structure (About page, feedback form etc). (Kasia)
Directly after pushing to production: emailing subscribers, inviting Report Coalition and Section Leaders (Kasia, Solana)

**Comment by kaodro** [comment]
Update: Translations should be ready on Aug 9 at latest so we can work towards Aug 9 as launch date but let's keep Aug 15 on the back of our heads in case we are delayed for other reasons (blogpost, emailing etc.)

**Comment by kaodro** [comment]
Hey, everything is going according to plan now and we should have all the staging site translated by tomorrow afternoon European time (CEST). When/What time can we expect it to be pushed to production tomorrow (Aug 9)? I will need the new URLs for the Emailing team that will be sending a massive emailing on Aug 10. Do you expect any problems? I can stand by all the evening on Aug 9 in case something goes wrong. 
Thanks! 
@kristinashu @cadecairos 
/cc @solanasaurus

**Comment by kristinashu** [comment]
Just confirming the key URLs will be:
- deploy https://healthreport.staging.wpengine.com/ to https://internethealthreport.org/
- end the redirect https://internethealthreport.org/ to https://internethealthreport.org/v01/
- keep https://internethealthreport.org/v01/ as is

@kaodro are you wanting to know the URL for the indicator blog post?

**Comment by kaodro** [comment]
@kristinashu that would be great, I could already submit it to Jessilyn for mailing. 
I would need a new one for the following blogposts:

https://healthreport.staging.wpengine.com/2017/08/04/here-is-how-well-measure-internet-health/
https://healthreport.staging.wpengine.com/es/2017/08/04/asi-mediremos-la-salud-de-internet/
https://healthreport.staging.wpengine.com/de/2017/08/06/so-werden-wir-die-internetgesundheit-messen/
The French one is still being translated.

https://healthreport.staging.wpengine.com/2017/05/15/how-would-you-measure-the-health-of-the-internet/
https://healthreport.staging.wpengine.com/fr/2017/05/15/comment-mesureriez-vous-letat-de-sante-dinternet/
https://healthreport.staging.wpengine.com/es/2017/05/15/como-se-puede-medir-la-salud-de-internet/
https://healthreport.staging.wpengine.com/de/2017/05/15/wie-wurden-sie-die-gesundheit-des-internets-messen/

Also if you could confirm the localized URLs for the main url:

https://healthreport.staging.wpengine.com/de/ = https://internethealthreport.org/de/
https://healthreport.staging.wpengine.com/es/ = https://internethealthreport.org/es/
https://healthreport.staging.wpengine.com/fr/ = https://internethealthreport.org/fr/

Thanks a million!

**Comment by cadecairos** [comment]
> Do you expect any problems?

I do expect there to be problems - WordPress multi-site installations are pretty error prone, especially when using WPEngines staging feature. 

My plan for this deployment is:

1. Take a backup of production
2. Using the production push tool in WPEngine, select the tables specific to the new site and have it copy them over
3. Test test test. If something is broken, I'll roll back.

When should I do this tomorrow? I'm in the EST timezone (6 hours behind CEST)

**Comment by kaodro** [comment]
@cadecairos We should be ready with all translations by 1 PM CEST = that means you could start early morning. If there will be alarming problems that would force us to postpone the launch, please let me know asap so I can let the emailing team know - they are preparing a big emailing linking to the website and blog content on it for Thursday, Aug 10.

**Comment by cadecairos** [comment]
okay, Could you signal here when it's time to start the push?

**Comment by kaodro** [comment]
Hey Cade, so far all is on track, I will let you know once all is ready to push in 2/3h. Just wanted to warn you that cause of the emailing (1,3 million people) we might receive lots of traffic tomorrow - is the website set up to handle that?
But no worries, ff anything goes wrong today, we are prepared to postpone the emailing to next week.
One more question: Mike will be fixing some minor bugs from tomorrow on (#16, #15, #14), should he be doing it on staging or directly in production? Also, if we want to change things (adjust posts, pages or post new ones), can we do it directly in production from today on, after the push? Thanks!

**Comment by kaodro** [comment]
@cadecairos the site is ready to be pushed! I will be on standby to test and also waiting for the final URLs to give to Jessilyn for the Emailing. Please keep me updated how it goes so I can flag her asap if we need to postpone. Thanks!
cc/ @kristinashu @solanasaurus

**Comment by cadecairos** [comment]
OK, I'm going to take backups of staging and production and get started. Are we certain no one is working on either staging or production sites right now? If they are, I can't guarantee their changes will be persisted if they make updates while I work on pushing the site.

**Comment by kaodro** [comment]
I just sent out an email to all involved to not work in wordpress cause you will be pushing it. Can you just quickly answer the questions from above? Crossing fingers for a smooth push!

**Comment by cadecairos** [comment]
> we might receive lots of traffic tomorrow - is the website set up to handle that?

yes, WPEngine handles scaling for us and we should be fine handling large amounts of traffic.



> should he be doing it on staging or directly in production?

I can deploy the theme update directly to Production.

> if we want to change things (adjust posts, pages or post new ones), can we do it directly in production from today on, after the push?

yes, once the site is live, we don't need to use staging for it anymore.

**Comment by cadecairos** [comment]
Updating this issue:

The new site is live at https://internethealthreport.org
v0.1 of the report continues to be available at https://internethealthreport.org/v01

Everything went over without much trouble - the only issues were that I needed to add the CDN url to the new landing site's Content Security Policy, and the new site structure changed the expected upload locations of files for the v01 report site. I fixed that by moving all the files from the old directories in the to new ones.

**Comment by kristinashu** [comment]
Awesome job everyone, thank you!
